src/contrib/cluster/Akka.Cluster.Metrics.Tests.MultiNode/ClusterMetricsExtensionSpec.cs

```diff
@@ -0,0 +1,164 @@
+// //-----------------------------------------------------------------------
+// // <copyright file="ClusterMetricsExtensionSpec.cs" company="Akka.NET Project">
+// //     Copyright (C) 2009-2020 Lightbend Inc. <http://www.lightbend.com>
+// //     Copyright (C) 2013-2020 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// // </copyright>
+// //-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+using Akka.Cluster.Metrics.Tests.Helpers;
+using Akka.Cluster.TestKit;
+using Akka.Configuration;
+using Akka.Remote.TestKit;
+using FluentAssertions;
+
+namespace Akka.Cluster.Metrics.Tests.MultiNode
+{
+    public class ClusterMetricsCommonConfig : MultiNodeConfig
+    {
+        public readonly RoleName Node1;
+        public readonly RoleName Node2;
+        public readonly RoleName Node3;
+        public readonly RoleName Node4;
+        public readonly RoleName Node5;
+
+        public IImmutableSet<RoleName> NodeList => ImmutableHashSet.Create(Node1, Node2, Node3, Node4, Node5);
```

Comment:
Ah, this is your bug.... The hashset may not list these in a consistent order.

Reply:
Ahh.... Exactly... They are unordered here... Missed this... Thanks!!